static/styles/recent_topics.css

```diff
@@ -72,10 +72,16 @@
         .table_fix_head table {
             /* To keep border properties to the thead th. */
             border-collapse: separate;
-            /* 440px = The amount of margin it takes to keep last topic row
-             * above maximum height of compose box.
+            /* For visual reasons, in a message feed, we require a large
+             * bottom_whitespace to make it convenient to display new
+             * messages as they come in and prevent occluding the last
+             * message with an open compose box. Here, the bottom item
+             * is rarely interesting context for a message one is
+             * composing, but we do need at least 41px to allow the
+             * close-compose-box UI element (including border) to not
+             * overlap content.
              */
-            margin-bottom: 440px;
+            margin-bottom: 50px;
         }
```

Comment:
I like the visuals/UX with this change better (i.e. no bottom_whitespace), but I guess I haven't tried the situation where we end up loading additional content -- is that what motivated having more here?

(I suppose we might want a `bottom_whitespace` type thing that's an extra 50px to hold a loading indicator, like we have at the bottom of message lists; we should at least open an issue for that as a follow-up)

Reply:
>  is that what motivated having more here?

I just though users may complain about not being able to see topic rows when compose box is open.

> (I suppose we might want a bottom_whitespace type thing that's an extra 50px to hold a loading indicator, like we have at the bottom of message lists; we should at least open an issue for that as a follow-up)

But we don't have a loading indicator in recent topics.

Reply:
I know we don't currently have a loading indicator there, but we very likely should add one in the future.  (I'm fine with leaving no space for now and just having a plan to add it when we do that; but it could also be nice to have blank row as a signal to the user that they've reached the end).

Reply:
Okay.